### ISSUE TITLE
Test Api Integration

### ISSUE THREAD

**Issue by arivera12** [sigaloid/LibreTranslate.Net#7]
- [x] Test api
  - [x]  Supported Languages
  - [x]  Translate

**Comment by arivera12** [comment]
I installed libretranslate using docker.
![image](https://user-images.githubusercontent.com/5600102/120852231-ef1df580-c547-11eb-8f2e-2c78f7f47b58.png)

Tested the api and here is the result.
![image](https://user-images.githubusercontent.com/5600102/120852200-e3caca00-c547-11eb-88d4-6ba2b5c454ec.png)

If you have more time and wanna do a more rigorous testing it would be great, but everything should work as expected.
